spec/spec_helper.rb

```diff
@@ -18,5 +18,7 @@
   config.include Rack::Test::Methods
   config.raise_errors_for_deprecations!
 
+  config.warnings = true
```

Comment:
please set it to `false`, it adds many additional warnings, and deprecation warnings could be overseen
->  run with ruby 2.4 to see what I mean